### PR TITLE
[14.0][ADD] account_journal_activate_tracking

### PR DESCRIPTION
**Pull request by IriaAlonso**
Activate tracking on account journal

### CODE REVIEW

File: account_journal_activate_tracking/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from odoo import models, fields
+
+class AccountJournal(models.Model):
+    _inherit="account.journal"
+    
+    name = fields.Char(string='Journal Name', required=True, tracking=True)
```
Comment (dalonsod) [inline account_journal_activate_tracking/models/account_journal.py]: When adding a new attribute to an existing field, no need to repeat existing attributes, simply do this:
```suggestion
    name = fields.Char(tracking=True)
```
Odoo will add new field attribute and preserve existing ones.

File: account_journal_activate_tracking/models/account_journal.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    bank_account_id = fields.Many2one('res.partner.bank',
```
Comment (dalonsod) [inline account_journal_activate_tracking/models/account_journal.py]: The same as above

File: account_journal_activate_tracking/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+{
+    "name": "Account journal - activate tracking",
+    "summary": """
+        Activate tracking on account journal
```
Comment (dalonsod) [inline account_journal_activate_tracking/__manifest__.py]: Better this IMO (more precise)
```suggestion
        Activate tracking on some account journal fields
```

File: account_journal_activate_tracking/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "14.0.1.0.0",
+    "category": "Accounting/Accounting",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": ["account"],
+    "data": [
+        
+    ],
```
Comment (dalonsod) [inline account_journal_activate_tracking/__manifest__.py]: Remove empty section

File: account_journal_activate_tracking/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
```
Comment (dalonsod) [inline account_journal_activate_tracking/__manifest__.py]: Wrong year

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments

**Comment by IriaAlonso** [comment]
@dalonsod cambios hechos y corregidos, ya puedes revisarlos

File: account_journal_activate_tracking/models/account_journal.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+class AccountJournal(models.Model):
+    _inherit="account.journal"
+    
+    name = fields.Char(tracking=True)
+    bank_account_id = fields.Many2one('res.partner.bank',tracking=True)
```
Comment (dalonsod) [inline account_journal_activate_tracking/models/account_journal.py]: Is `comodel_name` really needed, does it work?
```suggestion
    bank_account_id = fields.Many2one(tracking=True)
```

**Comment by IriaAlonso** [comment]
it is true that the comodel_name is not necessary, it works the same way

**Comment by dalonsod** [comment]
@ChristianSantamaria could you install this in ARivTest environment? Once installed, I'll test and, if it works as expected, I'll add `index.html` and merge it.

**Comment by ChristianSantamaria** [comment]
@dalonsod Done  :+1: :+1:

**Comment by dalonsod** [comment]
OK, tested, it works as expected.

Anyway, requirements for this addon have changed (sorry, I misunderstood customer original problem):
* We don't actually need to track `bank_account_id` => remove it
* And we need to track `default_account_id`, `code` and `currency_id`.

@IriaAlonso could you make these changes?

**Comment by IriaAlonso** [comment]
@dalonsod done

**Comment by dalonsod** [comment]
Great, could you update this addon in Test @ChristianSantamaria ?

**Comment by ChristianSantamaria** [comment]
@dalonsod Done

**Review by dalonsod — APPROVED** [review]
Added `index.html`.

**Pull request merged**